WordPress/src/main/res/values/strings.xml

```diff
@@ -1700,6 +1700,7 @@
     <!-- menu text -->
     <string name="reader_menu_tags">Edit tags and sites</string>
     <string name="reader_menu_block_blog">Block this site</string>
+    <string name="reader_menu_report_post">Report this post</string>
```

Comment:
We need to let Lorenzo know that we changed/added a new string - I believe we don't run the localization task on betafix builds.

Reply:
ooh, I did not know this. Thank you!!